environment.yml

```
@@ -12,15 +12,15 @@ dependencies:
   - dask-jobqueue
   - distributed
   - esgf-pyclient >=0.3.1
-  - esmpy >=8.6.0  # github.com/SciTools-incubator/iris-esmf-regrid/pull/342
+  - esmpy
```

```suggestion
  - esmpy !=8.1.0
```

As it [used to be](https://github.com/ESMValGroup/ESMValCore/pull/2402/files#diff-9efd195f4e9bfb79ccd456a1d8370fafcc4bcb0b00ea3799222667d2ae818533L16)?

It's also the same pin as in `setup.py`.

---

good call!